lenovo/thinkpad/l13-yoga/default.nix

```diff
@@ -0,0 +1,15 @@
+{ nixos, lib, pkgs, config, stdenv, ... }:
+{
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop/acpi_call.nix
```

Comment:
After https://github.com/NixOS/nixos-hardware/pull/285 you can just import l13 instead.
Actually I would move this profile to `l13/yoga.nix` instead.

Reply:
Should I move x13-yoga/default.nix to x13/yoga.nix as well?

Reply:
I think we should actually always make a directory, so `l13/yoga/default.nix`.
`x13-yoga/default.nix` could be moved to `x13/yoga` but with a symlink `x13-yoga` -> `x13/yoga` for backwards compatibility.

Reply:
ok, I don't think there's a need for backwardcompability, since nearly everyone will use the nixosmodules in flake.nix, or not?

Reply:
I think the majority of nixos users are still using nix stable.

Reply:
Oh, sorry I forgot that it's possible to use nixos-hardware without flakes.